dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -93,7 +92,7 @@ static ClassDumper create(List<Path> jarPaths) throws IOException {
     checkArgument(
         unreadableFiles.isEmpty(), "Some jar files are not readable: %s", unreadableFiles);
 
-    return new ClassDumper(jarPaths, extensionClassLoader, mapJarToClasses(jarPaths));
+    return new ClassDumper(jarPaths, extensionClassLoader, mapJarToClassFileNames(jarPaths));
```

Comment:
These variable name change is to clarify whether its class name (such as `AppWidgetset`) or class file name (such as `WEB-INF.classes.AppWidgetset`).